### PR TITLE
improve docs of binary_accuracy

### PR DESCRIPTION
**Pull request by naoto0804**
This is PR for improving docs of functions/links. Related issue: #2182

Modified functions/links:
 - functions/evaluation/binary_accuracy

### CODE REVIEW

**Comment by naoto0804** [comment]
screenshot

**Comment by naoto0804** [comment]
screenshot

File: chainer/functions/evaluation/binary_accuracy.py [hunk 1]
```diff
@@ -36,16 +36,44 @@ def binary_accuracy(y, t):
...
-        y (Variable): Variable holding a matrix whose i-th element
-            indicates the score of positive at the i-th example.
+        y (:class:`~chainer.Variable` or :class:`numpy.ndarray` or \
+        :class:`cupy.ndarray`):
+            Array whose i-th element indicates the score of
+            positive at the i-th sample.
+            The prediction label :math:`\\hat t[i]` is ``1`` if
+            ``y[i] >= 0``, otherwise ``0``.
+
         t (Variable): Variable holding an int32 vector of ground truth labels.
```
Comment (keisuke-umezawa) [inline chainer/functions/evaluation/binary_accuracy.py]: It seems that ``t`` can also be ``np.ndarray`` or ``cupy.ndarray`` like ``y``.
Please add these types for  the documentation.

**Comment by naoto0804** [comment]
screenshot_2

**Comment by naoto0804** [comment]
Thank you for the review. I've fixed.

**Comment by naoto0804** [comment]
Thank you for the review. I've fixed.

**Comment by keisuke-umezawa** [comment]
jenkins, test this please

**Comment by naoto0804** [comment]
@keisuke-umezawa  This PR is not approved. Is it OK? or should I fix more?

File: chainer/functions/evaluation/binary_accuracy.py (hunk 1 above)
Comment (naoto0804) [reply chainer/functions/evaluation/binary_accuracy.py]: Fixed

**Comment by keisuke-umezawa** [comment]
@naoto0804 Sorry for late reply.

@mitmul Please merge this PR!

**Comment by keisuke-umezawa** [comment]
jenkins, test this please

**Comment by mitmul** [comment]
Jenkins, test this please

**Comment by mitmul** [comment]
Jenkins, test this please

**Comment by mitmul** [comment]
LGTM

**Pull request merged**